os/fs/driver/mtd/smart.c

```diff
@@ -1103,7 +1103,7 @@ static int smart_setsectorsize(FAR struct smart_struct_s *dev, uint16_t size)
 	}
 	
 #ifdef CONFIG_MTD_SMART_JOURNALING
-	/** Journal Sector is reserved at the last of smartfs partition, it doesn't use MTD Header.
+	/** Journal Sector is reserved at the last of jSmartFs partition, it doesn't use MTD Header.
```

Comment:
Changes like these "smartfs->jSmartFs" are intended but "smart->jSmartFs" should be avoided for now.